index.html

```diff
@@ -51,13 +51,13 @@ <h1 class="welcome-title">Welcome to Real Dev Squad</h1>
           <h3>Check out our Docs</h3>
         </div>
         <div class="cards">
-          <div class="card">
+          <div class="card" onclick="location.href='./faq.html';">
```

Comment:
instead of redirecting it inline, you should extract it in a separate function with input parameter as redirectURL
something like
```
function redirectTo(redirectURL) {
  window.location.href=url;
}
```

Reply:
> instead of redirecting it inline, you should extract it in a separate function with input parameter as redirectURL
> something like
> 
> ```
> function redirectTo(redirectURL) {
>   window.location.href=url;
> }
> ```

Okay